src/GitHub.VisualStudio/Views/Dialog/ForkRepositoryExecuteView.xaml

```diff
@@ -0,0 +1,86 @@
+             xmlns:sampleData="clr-namespace:GitHub.SampleData;assembly=GitHub.App"
+             xmlns:ui="https://github.com/github/VisualStudio"
+             Margin="8"
+             mc:Ignorable="d" d:DesignWidth="300" Height="315.179">
+
+  <d:DesignProperties.DataContext>
+    <sampleData:ForkRepositoryExecuteViewModelDesigner/>
+  </d:DesignProperties.DataContext>
+
+  <StackPanel>
+    <TextBlock TextWrapping="Wrap">
+      You're about to fork the 
+      <Hyperlink>
+        <Run Text="{Binding SourceRepository.Owner, Mode=OneWay}"/>/<Run Text="{Binding SourceRepository.Name, Mode=OneWay}"/>
+      </Hyperlink>
+      repository to
+      <Hyperlink>
+        <Run Text="{Binding DestinationRepository.Owner, Mode=OneWay}"/>/<Run Text="{Binding DestinationRepository.Name, Mode=OneWay}"/>
+      </Hyperlink>. This operation will:
+    </TextBlock>
+
+    <ItemsControl Margin="8,16,8,8">
+      <ItemsControl.Resources>
+        <Style x:Key="ItemBorder" TargetType="Border">
+          <Setter Property="Background" Value="#10000000"/>
+          <Setter Property="CornerRadius" Value="3"/>
+          <Setter Property="Margin" Value="4"/>
+          <Setter Property="Padding" Value="4,8"/>
+        </Style>
+        <Style TargetType="CheckBox">
+          <Setter Property="Margin" Value="0,1,6,0"/>
+          <Setter Property="VerticalAlignment" Value="Center"/>
+        </Style>
+      </ItemsControl.Resources>
+      <Border Style="{StaticResource ItemBorder}">
+        <DockPanel>
+          <CheckBox IsChecked="True" IsEnabled="False"/>
+          <TextBlock>Fork the repository</TextBlock>
+        </DockPanel>
+      </Border>
+      <Border Style="{StaticResource ItemBorder}">
+        <DockPanel>
+          <CheckBox IsChecked="{Binding UpdateOrigin}"/>
+          <TextBlock TextWrapping="Wrap">
+          Update your local repository's <Run Style="{DynamicResource {x:Static markdig:Styles.CodeStyleKey}}">origin</Run> to point to
+          <Hyperlink><Run Text="{Binding DestinationRepository.CloneUrl, Mode=OneWay}"/></Hyperlink>
+          </TextBlock>
+        </DockPanel>
+      </Border>
+      <Border Style="{StaticResource ItemBorder}">
+        <DockPanel>
+          <CheckBox IsChecked="{Binding AddUpstream}" IsEnabled="{Binding CanAddUpstream}" />
+          <TextBlock TextWrapping="Wrap">
+            Add an <Run Style="{DynamicResource {x:Static markdig:Styles.CodeStyleKey}}">upstream</Run> remote pointing to 
+          <Hyperlink><Run Text="{Binding SourceRepository.CloneUrl, Mode=OneWay}"/></Hyperlink>
+          </TextBlock>
+        </DockPanel>
+      </Border>
+      <Border Style="{StaticResource ItemBorder}">
+        <DockPanel>
+          <CheckBox IsChecked="{Binding ResetMasterTracking}" IsEnabled="{Binding CanResetMasterTracking}" />
+          <TextBlock TextWrapping="Wrap">
+            Set the <Run Style="{DynamicResource {x:Static markdig:Styles.CodeStyleKey}}">master</Run> branch to track
```

Comment:
It would be nice to have a tooltip saying why this is useful and how it would be used.

Reply:
I have no idea what to write in the tooltip. What do you think I should put?